src/mail/view/ConversationViewer.ts

```diff
@@ -86,7 +87,7 @@ export class ConversationViewer implements Component<ConversationViewerAttrs> {
 				this.renderLoadingState(viewModel),
 				this.renderFooter(),
 			),
-			this.renderFloatingSubject(),
+			client.isMobileDevice() ? null : this.renderFloatingSubject(),
```

Comment:
this should rather depend on single column layout like the rest of the mail layout

Reply:
That makes more sense. I had mobile stuck in my head, haha